Topics/User_Experience/Color_Theory.md

```diff
@@ -0,0 +1,45 @@
+* **Green**: evokes calming and renewing energy, as it is a color that symbolizes nature, growth, and harmony. Green also evokes envy and jealousy, as it is associated with notions of wealth and materialism. In design, the context should be considered when using green in combination with other colors.
+* **Blue**: evokes feelings of trust and reliability. In design, it can be used to convey trust but should be incorporated in moderation as it also associates with feelings of distance and sadness.
+* **Black**: has many meanings; it is associated with mystery, death, and can convey meanings of tradition, seriousness, or moderness. Context and color combinations determine the vibe it gives off. In design, black goes well with everything hence is largely used as background color. It is also often used to set contrasts.
+* **White**: has meanings of purity, innocence, and clarity. White can symbolize newness and motivate people to generate new ideas, yet too much can cause feelings of isolation and loneliness. In design, white is commonly used as a background color to enhance redability.
+
+### Nuances of Color Preferences
+While colors can take on general meanings as shown above, designers have to keep in mind that perception is bound to differ among individuals. In particular, factors such as age, gender, and culture can work to shift people's perceptions. 
+* **Age**: Mental and physical changes across our lifetime can shift our preferences and attraction to colors. For example, yellow is color that is liked by many children, but less shared among adults. Faber Birren also explains it in his work: “With maturity comes a greater liking for hues of shorter wavelength (blue, green, purple) than for hues of longer wavelength (red, orange, and yellow)”.
+* **Culture**: Cultures define and interpret colors differently. The meanings listed above are related to Western perceptions of color, which may differ from countries across the world. For example, while white may be associated with purity in Western cultures, it denotes connotations of death and mourning in numerous Asian cultures.
+* **Gender**: Different genders may favour different colors, a phenomenon whose roots can be traced back to evolutionary biology and psychology. For example, research has shown that females evolved to prefer pinkish/reddish color because of their role in foraging ripe fruits and caring for the young. 
+
+Other factors to consider are personal experience, mood, status and so on. The important point to recognize is that there are multiple forces acting on another that contribute to how an individual would perceive of a color, and is something that designers should keep in mind when making appropriate decisions about color branding. 
```

Comment:
I think here saying "something that designers should keep in mind when making appropriate decisions about color branding." is a bit vague. Maybe an improvement would be to say that it is import to understand the target user and tailor to that particular user group.  

You could use examples from existing products. (Mobile games can be a good source for this. E.g. Candy Crush)

Reply:
That's a good suggestion. Yes, I think I will incorporate interfaces from Candy Crush to help elucidate the point more clearly.